doc-Managing_Authentication/topics/External_auth_troubleshooting.adoc

```diff
@@ -0,0 +1,40 @@
+[[external_troubleshooting]]
+= Troubleshooting External Authentication Configuration
+
+////
+ Is this needed? When do you need this?
+Using the appliance_console to generate External HTTP Authentication generates the following files:
+
+/etc/httpd/conf.d/manageiq-external-auth.conf
+/etc/sssd/sssd.conf
+With the above configuration files in place, HTTP will query, using dbus, SSSD which will perform the LDAP/S authentication and authorization steps necessary to validate users and groups.
```

Comment:
A quick online search shows me more instances of "D-Bus". Thought not sure if there's one correct way  - perhaps worth checking the IBM style guide?

Reply:
Not in IBM, but Googling makes me agree with D-Bus. I've changed this (though it is commented out as I wasn't sure if it was needed)